lib/senders/sms.js

```diff
@@ -84,17 +84,28 @@ module.exports = function (log, translator, templates, smsConfig) {
     return P.promisify(object[methodName], { context: object })
   }
 
-  function getMessage (templateName, acceptLanguage) {
+  function getMessage (templateName, acceptLanguage, signinCode) {
     var template = templates['sms.' + templateName]
 
     if (! template) {
       log.error({ op: 'sms.getMessage.error', templateName: templateName })
       throw error.invalidMessageId()
     }
 
+    var link
```

Comment:
Would it simplify the code a bit to pass in the link instead of a signinCode and create the link in lib/routes/sms.js where the `features` parameter is ingested and the signinCode created? Seems like at a minimum it would localize cause & effect of the `features` parameter.